src/css/styles.css

```diff
@@ -13,61 +12,211 @@
+    background: #84ffff;
+    grid-area: main;
+    border-radius: 10px;
+    padding-top: 5px;
+}
+
+.content1 {
+    background: #6fffd2;
+    grid-area: content1;
+    border-radius: 10px;
+    padding-top: 5px;
+}
+
+.content2 {
+    background: #64ffda;
+    grid-area: content2;
+    border-radius: 10px;
+    padding-top: 5px;
+}
 
 /* *** Header *** */
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+img {
+    width: 50px;
+    padding: 5px;
+}
+
 .header__left {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+}
 
+.header__left--text {
+    font-family: 'Alice', serif;
+    font-size: 1.5rem;
+    padding: 5px;
 }
 
 .header__right--text {
     font-family: 'Alice', serif;
+    font-size: 1.5rem;
+    padding: 5px;
 }
 
 /* *** Navigation *** */
-img {
-    width: 70px;
-}
-.nav__left--text {
-    font-family: 'Alice', serif;
+.nav__right {
+    padding-top: 10px;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
 }
 
 .nav-links {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
     list-style-type: none;
 }
 
+.nav-links__item {
+    flex-wrap: nowrap;
+    margin: 0 20px;
+}
+
 /* *** Main *** */
 main {
     border: 1px solid lightblue;
-    padding: 50px;
+    padding: 20px;
+}
+
+/* *** Recipe detail *** */
+.recipe-detail__top {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: center;
 }
 
 /* *** Ingredients *** */
+.pantry__top {
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-between;
+    align-items: center;
+}
+
+/* *** Recipe list *** */
+.recipe {
+    margin: 10px 5px;
+    display: flex;
+    flex-flow: row nowrap;
+    justify-content: space-evenly;
+    align-items: flex-start;
+}
+
+.item-container {
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: space-around;
+    align-items: center;
+}
+
+.ingredients-and-cost {
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: flex-start;
+    align-items: flex-start;
+
+}
+
+ul {
+    list-style-type: none;
+}
 
 /* *** Footer *** */
 footer {
-    border-top-left-radius: 7px;
-    border-top-right-radius: 7px;
     background: #CBBDBE;
-    padding: 10px;
+    grid-area: footer;
+    border-radius: 10px;
+    padding-top: 5px;
 }
 
 .ingredients__message {
     font-size: 0.6em;
     color: red;
 }
 
+/* *** miscellaneous *** */
 .hidden {
     display: none;
 }
 
 /* *** breakpoint 800px *** */
 @media screen and (max-width: 800px) {
-    .nav-links {
+    .nav__left--logo {
+        display: none;
+    }
+
+    .grid-container {
+        grid-template-columns: .1fr 1.9fr;
+        grid-template-rows: .4fr .5fr 2.5fr 2.5fr .4fr;
+        grid-template-areas:
+            "header header"
+            "nav main"
+            "nav content1"
+            "nav content2"
+            "footer footer";
+    }
```

Comment:
Cool seeing how the nav switches from horizontal to vertical when switching from desktop to mobile view. Great work!